tests/pages/api/annotationAPI.js

```diff
@@ -0,0 +1,43 @@
+const { I } = inject();
+const assert = require('assert');
+
+/* eslint-disable consistent-return */
```

Comment:
why?

Reply:
It's unimportant after my fix. Deleted! :)